eng/testing/AppleRunnerTemplate.sh

```diff
@@ -56,7 +56,15 @@ else
     export XHARNESS_OUT="$HELIX_WORKITEM_UPLOAD_ROOT/xharness-output"
 fi
 
-dotnet xharness ios test  \
+if [ ! -z "$XHARNESS_CLI_PATH" ]; then
```

Comment:
So this script is used in CI?

Reply:
It is for WASM where I copied this code block from 😄 
Looks like Android has it too even though it's not true there...

We should look at reusing the script logic from your arcade helix sdk like we talked about a while ago.

Reply:
The script in Helix SDK has several different functions by now so I am not sure it is really what we want:
- accept env vars because of `launchctl`
- calling `ios test`/`ios run` command based on arguments
- collecting logs and result XML and placing them in the Helix upload folder
- rebooting the machine / retry of the work item when install hangs
- disabling colors / enabling timestamps for logs
- (soon) signing the app using the Keychain present on Helix agents

Many of these steps are probably not desired for local flows. Recently, I even split the script into 2 to fix the retry/reboot flow. Overall I consider the script to be very Helix-infra-agnostic so it really only makes sense to use it inside Helix jobs

Reply:
Ok then I guess I'll instead put a notice at the top of the AppleRunnerTemplate.sh that it isn't used in CI.

Reply:
That's a good idea! There was quite a lot of confusion around this already.

Also afaik Android might get its own script in Helix SDK soon too. I think it's how Matt will reboot the emulators